jdk_11_maven/em/embedded/rest/market/src/main/java/em/embedded/market/EmbeddedEvoMasterController.java

```diff
@@ -138,7 +138,8 @@ public SutInfoDto.OutputFormat getPreferredOutputFormat() {
     public List<AuthenticationDto> getInfoForAuthentication() {
```

Comment:
need to update `ExternalEvoMasterController` as well

Reply:
@onurd86  btw, I assume you then run EvoMaster on this update settings, and verified there was no message saying 401 with this new user, isn't it?

Reply:
Hi Andrea,

Thanks for mentioning that. I tried to run EvoMaster again after adding the new user. I do not get any 401 after adding the new user. Adding a new user to market was very easy.

Reply:
I have updated External Controller as well.

Reply:
This was one of the successful tests generated by EvoMaster after adding user2:

 ```
@Test @Timeout(60)
    public void test_1() throws Exception {
        
        given().accept("*/*")
                .header("Authorization", "Basic dXNlcjJAeWFuZGV4LnJ1Onl1cmlkb2xnb3J1a2k=") // user2
                .header("x-EMextraHeader123", "")
                .get(baseUrlOfSut + "/customer/orders?name=ATX")
                .then()
                .statusCode(200)
                .assertThat()
                .contentType("application/json")
                .body("size()", equalTo(0));
    }
```